rift/node.py

```diff
@@ -1858,9 +1858,14 @@ def store_tie_packet_info(self, tie_packet_info):
             reason = "TIE " + packet_common.tie_id_str(tie_id) + " added"
         self.tie_packet_infos[tie_id] = tie_packet_info
         if self.is_same_level_tie(tie_packet):
-            self.peer_node_tie_packet_infos[tie_packet.header.tieid] = tie_packet_info
-            self.update_partially_conn_all_intfs()
-            self.regenerate_my_south_prefix_tie()
+            # Ignore E-W links as peer neighbors (only southern reflected TIEs should be considered)
+            # TODO: Ask to Bruno
+            ew_neigh = dict(self.node_neighbors(self.node_ties(constants.DIR_SOUTH, self.system_id),
```

Comment:
We added this additional check when storing the peer nodes, since they was causing problems when ToF rings were used.